tests/runtime/test_grpc_server.py

```diff
@@ -1350,6 +1351,63 @@ def test_grpc_sever_shutdown_with_model_poll(open_port):
         assert not server_proc.killed
 
 
+def test_all_signal_handlers_invoked(open_port):
+    """Test that a SIGINT successfully shuts down all running servers"""
+
+    # whoops, need 2 ports. Cannot use `_open_port()` again because it'll return the same port
+    other_open_port = open_port + 37
+
+    with tempfile.TemporaryDirectory() as workdir:
+        server_proc = ModuleSubproc(
+            "caikit.runtime",
+            kill_timeout=30.0,
+            RUNTIME_GRPC_PORT=str(open_port),
```

Comment:
Since this is testing with logs (meh), throw a `LOG_LEVEL` in there so that we guarantee the logs will be configured to print in case the parent invokes the test with logging turned all the way off

Reply:
hot